### PR TITLE
rebrand to lowercase web3.py

### PR DESCRIPTION
**Pull request by wolovim**
### What was wrong?

the capital W is decidedly not hip; needed rectifying immediately

### Todo:

- [x] Add entry to the [release notes](https://github.com/ethereum/web3.py/blob/master/newsfragments/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Web3.py
+# web3.py
 
 [![Documentation Status](https://readthedocs.org/projects/web3py/badge/?version=latest)](https://web3py.readthedocs.io/en/latest/?badge=latest)
 [![Discord](https://img.shields.io/discord/809793915578089484?color=blue&label=chat&logo=discord&logoColor=white)](https://discord.gg/GHryRvPB84)
 [![Build Status](https://circleci.com/gh/ethereum/web3.py.svg?style=shield)](https://circleci.com/gh/ethereum/web3.py)
 
-A Python library for interacting with Ethereum, inspired by [web3.js](https://github.com/ethereum/web3.js).
```
Comment (wolovim) [inline README.md:7]: for posterity: indeed, original inspiration from web3.js (:salute:), but in the ongoing development of web3.py, that lib is no longer referenced on any regular basis, at least not any more than ethers or another language implementation.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+A Python library for interacting with Ethereum.
```
Comment (wolovim) [inline README.md:7]: for posterity: indeed, original inspiration from web3.js (:salute:), but in the ongoing development of web3.py, that lib is no longer referenced on any regular basis, at least not any more than ethers or another language implementation.

File: docs/conf.py [hunk 3]
```diff
@@ -52,7 +52,7 @@
 master_doc = "index"
 
 # General information about the project.
-project = "Web3.py"
+project = "web3.py"
 copyright = "2018, Piper Merriam, Jason Carver"
```
Comment (kclowes) [inline docs/conf.py:56]: Should we update this too? At least the date, not sure about the names?

**Review by kclowes — APPROVED** [review]
LGTM! I didn't realize `Web3.py` had been so standardized.

File: docs/conf.py (hunk 3 above)
Comment (fselmo) [reply docs/conf.py:56]: +1... looks like we'd been moving towards "YYYY, The Ethereum Foundation" in all the other libraries.

**Review by fselmo — APPROVED** [review]
lgtm 👍🏼 

I found [one more](https://github.com/ethereum/web3.py/blob/6a7de517f794e2591441239030fcdde8fee2993c/docs/_templates/searchbox.html#L20) if you want tiny nit in your life 😆

File: docs/conf.py (hunk 3 above)
Comment (wolovim) [reply docs/conf.py]: official foundation site copyright left off the "The", so i did too 🤝

**Pull request merged**